.github/workflows/python-ci-packaging.yml

```diff
@@ -351,6 +353,7 @@ jobs:
     steps:
       - uses: actions/checkout@v4
         with:
+          path: TileDB-SOMA
           fetch-depth: 0 # for setuptools-scm
```

Comment:
Just noting, `fetch-depth: 0` shouldn't be necessary anymore (since #2563). Fine to leave it in as well.

Reply:
I agree it isn't strictly necessary. But I want to keep this test job as close to as possible to the upstream official packaging job that it is designed to emulate

https://github.com/single-cell-data/TileDB-SOMA/blob/6108f733e1d616f20a285a92a8695f124a795d44/.github/workflows/python-packaging.yml#L27-L30